canonical-websites/www.ubuntu.com
# update /core/contact-us to vbt

## Done

* updated /core/contact-us to VBT

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/core/contact-us)
- [demo](http://www.ubuntu.com-1599-core-contact-us.demo.haus/core/contact-us)


## Issue / Card

Fixes #1599
